### ISSUE TITLE
Console user password

### ISSUE THREAD

**Issue by pakoto** [matheusjgsantos/MSXPiHat-OS#1]
Can´t access to console using F12 - Alt F2. Try root no password, raspberry, pi, etc.

Please share user password

**Comment by matheusjgsantos** [comment]
Root's password is fudeba
You can change this on the .config file also.

**Comment by pakoto** [comment]
Thanks! Now i can accesss. I´m trying to solve cut sound with raspi3.

**Comment by matheusjgsantos** [comment]
Check if this parameter is set on the config.txt file:
dtparam=audio=on

**Comment by pakoto** [comment]
thanks. I suppose that nano not included. Any easy option to modify/copy files?

**Comment by matheusjgsantos** [comment]
Sorry  but I didn't included nano, so you will need to use vi. I will remember to add nano on the next version.
If you want to change the config.txt file you can do this on Windows os Mac desktop/notebook, since it's on the FAT partition and you just need to mount the SD card to have access to it.

**Comment by pakoto** [comment]
Thanks. No problem with vi. I see that dtparam audio still on. I´m using msxpihat 0.5 image. Testing compiled bluemsx-pi file included into meesokim github with raspbian sounds ok. I´m trying to replace it into your image.

**Comment by pakoto** [comment]
Do you test meesokim openmsx? Compiled but don´t work. Tested with raspbian lite

**Comment by matheusjgsantos** [comment]
Openmsx requires Python2 which is not available on buildroot. I'm working on a Yocto image with Python to test openmsx,  but Yocto take a really long time to build. I wish I could do a bare bone openmsx or bluemsx implementation, that would be perfect.

**Comment by pakoto** [comment]
yeah. Barebone msx emu would be a dream. Could you add last blueberrymsx version to msxpihat 0.5 image?

**Comment by pakoto** [comment]
http://zxmini.speccy.org/en/index.html

https://github.com/rsta2/circle
